### DISCUSSION TITLE
problem with BMW Conneted Drive in Home assistant !

### DISCUSSION THREAD

**Discussion by schumi2309** [bimmerconnected/bimmer_connected#641] [Q&A]
Good morning,
I have a BMW 318d from 2018 and I just installed the BMW Connected Drive integration. after having entered my login and password, it found 6 entities. but the problem is that everything remains “unknown”.
Is there anything more I need to do after installing the integration? why doesn't this work?
thank you.

**Comment by rikroe** [comment]
Continuing this in #642
